### PR TITLE
Added guild log stash/treasury/members to guild log command

### PR DESCRIPTION
**Pull request by n1tr0-5urf3r**

### CODE REVIEW

File: guildwars2/guild/general.py [hunk 1]
```diff
@@ -174,60 +174,100 @@ class GeneralGuild:
...
-            guild_id = guild["id"]
-            guild_name = guild["name"]
-            endpoint = "guild/{0}/log/".format(guild_id)
-            log = await self.call_api(endpoint, ctx.author, ["guilds"])
-        except (IndexError, APINotFound):
-            return await ctx.send("Invalid guild name")
-        except APIForbidden:
+        state = log_type.lower()
+        member_list = ["invited", "joined", "invite_declined"]
+        if state == "stash" or state == "treasury" or state == "members":
```
Comment (Maselkov) [inline guildwars2/guild/general.py]:
```py
if state not in ("stash", "treasury", "members"):
    return "some help message"

File: guildwars2/guild/general.py [hunk 2]
```diff
@@ -174,60 +174,100 @@ class GeneralGuild:
...
+                return await self.error_handler(ctx, e)
+        else:
             return await ctx.send(
-                "You don't have enough permissions in game to "
-                "use this command")
-        except APIError as e:
-            return await self.error_handler(ctx, e)
-        data = discord.Embed(description="Stash Log", colour=self.embed_color)
+                "{0.mention}, Please us either 'stash' or 'treasury' as parameter".
+                    format(ctx.author))
```
Comment (Maselkov) [inline guildwars2/guild/general.py]: should provide a command syntax example

File: guildwars2/guild/general.py [hunk 3]
```diff
@@ -174,60 +174,100 @@ class GeneralGuild:
...
+                            value=user + "{} {}{} {}".format(
+                                operator, quantity, multiplier, item_name),
+                            inline=False)
+                        counter += 1
+            if state == "members":
+                if entry["type"] in member_list:
+                    time = entry["time"]
+                    timedate = datetime.datetime.strptime(
+                        time,
+                        "%Y-%m-%dT%H:%M:%S.%fZ").strftime('%d.%m.%Y %H:%M')
```
Comment (Maselkov) [inline guildwars2/guild/general.py]: i have a feeling theres an easier way to do this

File: guildwars2/guild/general.py (hunk 1 above)
Comment (n1tr0-5urf3r) [reply guildwars2/guild/general.py]: Changed

File: guildwars2/guild/general.py (hunk 2 above)
Comment (n1tr0-5urf3r) [reply guildwars2/guild/general.py]: Changed to "return await self.bot.send_cmd_help(ctx)"

**Pull request merged**